layouts/default.vue

```diff
@@ -1,5 +1,18 @@
 <template>
-  <div>
-    <Nuxt />
+  <div class="container mx-auto m-0 p-2 h-screen max-w-4xl">
+    <div class="flex m-4 justify-around">
+      <NuxtLink :to="`/`">
+        <img src="~/static/eucp_logo.png" alt="EUCP Logo">
+      </NuxtLink>
+      <div class="flex m-4 justify-around items-center">
+        <NuxtLink :to="`/`">
+          <p class="text-sm text-gray-400">
+            Home
+          </p>
+        </NuxtLink>
+          <input class="w-32 h-8 focus:shadow-outline rounded text-sm px-2 pl-4" type="search" placeholder="Search...">
+      </div>
+    </div>
+      <Nuxt />
```

Comment:
See the linter warnings. Did you set up your editor to do some nice auto-formatting with eslint? Otherwise I highly recommend you do that, I can help if you want.

Reply:
thanks for the tips, I am using it now.